delete_topics_request.go

```diff
@@ -42,6 +42,10 @@ func (d *DeleteTopicsRequest) headerVersion() int16 {
 	return 1
 }
 
+func (d *DeleteTopicsRequest) isValidVersion() bool {
+	return d.Version >= 0 && d.Version <= 1
```

Comment:
Versions 0, 1, 2, and 3 are the same and the responses only include semantic and error response differences?

```suggestion
	return d.Version >= 0 && d.Version <= 3
```